Assignment1/body_mass_index_calculator.py

```diff
@@ -0,0 +1,36 @@
+class BodyMassIndexCalculator:
+
+    # https://www.nhlbi.nih.gov/health/educational/lose_wt/BMI/bmicalc.htm
+
+    @staticmethod
+    def convert_weight(weight):
```

Comment:
I'm not sure what this function does. I think it's pounds to kilograms, but I don't actually know. This and the other conversion functions could be more explicitly named.